src/api/routers/shift.py

```diff
@@ -88,11 +79,7 @@ async def get_shift(
         status_code=HTTPStatus.OK,
         summary="Обновить информацию о смене",
         response_description="Обновленная информация о смене",
-        responses={
-            400: ERROR_TEMPLATE_FOR_400,
-            403: ERROR_TEMPLATE_FOR_403,
-            404: ERROR_TEMPLATE_FOR_404,
-        },
+        responses=generate_error_responses(400, 404),

```

Comment:
В таком варианте сложнее совершить ошибку, и номера статусов не нужно запоминать, чтобы понять, какая ошибка может возвращаться. Сейчас используются "популярные" статусы, а если нужно будет добавить что-то специфическое, то так не нужно смотреть в справочнике, а подобрать по названию.  
```suggestion
        responses=generate_error_responses(HTTPStatus.BAD_REQUEST, HTTPStatus.NOT_FOUND),
```